python/pyarrow/_fs.pyx

```diff
@@ -1027,6 +1027,23 @@ cdef class PyFileSystem(FileSystem):
     ----------
     handler : FileSystemHandler
         The handler object implementing custom filesystem behavior.
+
+    Examples
+    --------
+    Create an fsspec-based filesystem object for GitHub:
+
+    >>> from fsspec.implementations import github
+    >>> gfs = github.GithubFileSystem('apache', 'arrow')
+
+    Get a PyArrow FileSystem object:
+
+    >>> from pyarrow.fs import PyFileSystem, FSSpecHandler
+    >>> pa_fs = PyFileSystem(FSSpecHandler(gfs))
+
+    Use :func:`~pyarrow.fs.FileSystem` functionality `get_file_info()`:
```

Comment:
```suggestion
    Use :func:`~pyarrow.fs.FileSystem` functionality ``get_file_info()``:
```

Reply:
I can't seem to remember to use double `` 🤦‍♀️  Thanks!